build.gradle

```diff
@@ -142,19 +145,55 @@ tasks.withType(JavaCompile).configureEach {
 	options.compilerArgs.add '-XDstringConcat=inline'
 }
 
-build.dependsOn('installLocalGitHookPostCheckout')
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-	group = 'git'
-	from new File(rootProject.rootDir, '.scripts/post-checkout')
-	into { new File(rootProject.rootDir, '.git/hooks') }
-	filePermissions {
-		unix(0775)
+
+// Setup local git hooks
+
+def ensureSymlink(link, target) {
+	if (!Files.isSymbolicLink(link)) {
+		Files.deleteIfExists(link)
+
+		Files.createSymbolicLink(link, target)
+
+		def relative = { f -> rootDir.toPath().relativize(f) }
+		println("--- Installed symbolic link ${relative(link)} -> ${relative(target)} ---")
 	}
+}
+
+def gitConfig = 'Git Configuration'
```

Comment:
```suggestion
def gitConfigGroupName = 'Git Configuration'
```

Reply:
I'm pretty sure it's more idiomatic to do it as I did. From usage, `group = gitConfig` is of course much nicer than `group = gitConfigGroupName`